doc/TYPES2021/Demos/HSP.tex

```diff
@@ -1822,12 +1714,11 @@ \subsection{Algebras}\label{algebras}
 \ab{α} \ab{ρᵃ} whereas \agda expected an inhabitant of the type \ar{Algebra} (\ab{𝓞}
 \aop{⊔} \ab{𝓥} \aop{⊔} (\ap{lsuc} \ab{α})) \ab{ρᵃ}.
 \fi
-To resolve such problems, we use the \AgdaRecord{Lift} record type of the \agdastdlib,
-which takes a type inhabiting a particular universe and embeds it into a higher universe.
-Specializing the \ar{Lift} type to our domain of interest, the \agdaalgebras library
-defines a function called \af{Lift-Alg}%
+\agda provides \ar{Lift} to embed a type into a higher universe.
```

Comment:
Come to think of it, I now recall spelling this out in more detail because one referee of the old version complained that I hadn't explained what Lift does! :)

Reply:
It's okay as is.